ide/coqide/coqOps.ml

```diff
@@ -578,6 +609,19 @@ object(self)
           Minilib.log ("Unsupported feedback message " ^ mtype ^ " with a sentence")
       in
       let log ?id s = log_pp ?id (Pp.str s) in
+      (* When a command generates a warning or error, the message sent has an
+         incorrect sentence id.  If bp..ep is out of range for the specified
+         sentence, assume is it is from the last-sent "add" request *)
```

Comment:
How come the sentence id is wrong, you mean it lack the Id?

Reply:
AFAIK every feedback message includes a sentence id (`state_id` below), however if CoqIDE doesn't have that state id in its document, it's considered unknown.  I found that both this and the code at 592-600 was needed for differing use cases.  I would have repeat some testing to describe exactly what was happening.

```
<feedback object="state" route="0">
  <state_id val="2"/>
  <feedback_content val="message">
    <message>
      <message_level val="warning"/>
      <option val="some">
        <loc start="11" stop="16"/>
      </option>
```